### PR TITLE
Fix Tags Field in CreateStreamRequest Caused ValidationException

### PR DESCRIPTION
**Pull request by zhiyua-git**
*Issue #, if available:*
https://github.com/aws-amplify/aws-sdk-android/issues/1065

*Description of changes:*
Set the tags value to null so Kinesis Video will not throw ValidationException on this.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: aws-android-sdk-kinesisvideo/src/main/java/com/amazonaws/mobileconnectors/kinesisvideo/service/KinesisVideoAndroidServiceClient.java [hunk 1]
```diff
@@ -192,7 +192,8 @@ public String createStream(@NonNull final String streamName,
                 .withDeviceName(deviceName)
                 .withMediaType(contentType)
                 .withKmsKeyId(StringUtils.isBlank(kmsKeyId) ? null : kmsKeyId)
-                .withDataRetentionInHours((int) retentionPeriodInHours);
+                .withDataRetentionInHours((int) retentionPeriodInHours)
+                .withTags(null);
```
Comment (kvasukib) [inline aws-android-sdk-kinesisvideo/src/main/java/com/amazonaws/mobileconnectors/kinesisvideo/service/KinesisVideoAndroidServiceClient.java]: Will there be a case where tags can be passed? Can the developer override this?

Comment (zhiyua-git) [reply aws-android-sdk-kinesisvideo/src/main/java/com/amazonaws/mobileconnectors/kinesisvideo/service/KinesisVideoAndroidServiceClient.java]: No. It is not expected to be passed on creation now. Passing the tags on creation is complex change also related to SDK state machine change. This feature might be support later.

Comment (kvasukib) [reply aws-android-sdk-kinesisvideo/src/main/java/com/amazonaws/mobileconnectors/kinesisvideo/service/KinesisVideoAndroidServiceClient.java]: Can you add a comment in the code to explain why tags are passed as null?

Comment (kvasukib) [reply aws-android-sdk-kinesisvideo/src/main/java/com/amazonaws/mobileconnectors/kinesisvideo/service/KinesisVideoAndroidServiceClient.java]: Also, please add a statement in CHANGELOG.md file as a bug fix.

Comment (zhiyua-git) [reply aws-android-sdk-kinesisvideo/src/main/java/com/amazonaws/mobileconnectors/kinesisvideo/service/KinesisVideoAndroidServiceClient.java]: Updated.

**Pull request merged**